include/zephyr/drivers/usb_c/usbc_pd.h

```diff
@@ -53,6 +53,93 @@ extern "C" {
  */
 #define PD_MAX_EXTENDED_MSG_CHUNK_LEN    26
 
+/**
+ * @name USB PD 3.1 Rev 1.6, Table 6-70 Counter Parameters
+ * @{
+ */
+
+/**
+ * @brief The CapsCounter is used to count the number of Source_Capabilities
+ *	  Messages which have been sent by a Source at power up or after a
+ *	  Hard Reset.
+ *	  Parameter Name: CapsCounter
```

Comment:
Non-blocking comment.

The parameter name is `nCapsCount`.  `CapsCounter` is the "counter" name from table 6-71.  Probably better to use `nCapsCount` since that is the name listed in table 6-70.

Reply:
done